exonum/tests/config.rs

```diff
@@ -200,14 +210,17 @@ fn test_generate_template() {
     }
 }
 
-#[test]
 #[cfg_attr(feature = "cargo-clippy", allow(needless_range_loop))]
```

Comment:
Is this attribute really needed here? The `needless_range_loop` lint doesn't seem to be triggered by the test code. Btw, the legitimate triggering that occurs in the `finalize_config` function could be avoided by rewriting the `for` loop in it as

```rust
for &conf in PUB_CONFIG.iter().take(count) {
    override_validators_count(conf, count, folder);
    variables.push(full_tmp_name(conf, folder));
}
```